### PR TITLE
restore system config after running acceptance test

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

Restore system config after running acceptance test scenarios
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Review by skshetry — COMMENTED** [review]
LGTM :slightly_smiling_face:

File: tests/acceptance/features/bootstrap/AppConfiguration.php [hunk 1]
```diff
@@ -457,6 +457,21 @@ private function restoreParameters($server) {
...
+		foreach ($this->savedConfigList[$server]['system'] as $configKey => $configValue) {
+			if (!\array_key_exists($configKey, $currentConfigList["system"])
+				|| $currentConfigList["system"][$configKey] !== $this->savedConfigList[$server]['system'][$configKey]
+			) {
+				if (\gettype($configValue) === "string") {
+					SetupHelper::runOcc(
+						['config:system:set', "--value $configValue", $configKey]
+					);
+				} elseif (\gettype($configValue) === "array") {
+					SetupHelper::runOcc(
```
Comment (individual-it) [inline tests/acceptance/features/bootstrap/AppConfiguration.php]: try to run that for any case, `type=json` should work for any type.
currently you have e.g. not covered `boolean`

**Review by individual-it — CHANGES_REQUESTED** [review]

**Comment by skshetry** [comment]
@ownclouders rebase

**Review by individual-it — APPROVED** [review]

**Comment by HariBhandari07** [comment]
Backport on #35141

**Pull request merged**